srml/council/src/lib.rs

```diff
@@ -14,7 +14,221 @@
+//! weight of the _presented_ candidates.
+//! Based on the configurations of the module, `N` top candidates in the leaderboard are immediately recognized
+//! as councillors (where `N` is `desired_seats - active_council.len()`) and `C` of the remaining are kept in
+//! the leaderboard as _carry_ for the next tally to compete again. Similarly, councillors whose term has ended
+//! are also removed.
+//!
+//! - **Vote index**: A counter pointing the the number of tally rounds already applied.
+//! - **Desired seats:** The number of seats on the council.
+//! - **Candidacy bond:** Bond required to be a candidate. The bond is returned to all candidates at the end
+//! of the tally if they have won the tally (e.g. have become a councillor).
+//! - **Leaderboard:** A list of candidates and their respective votes in an election, ordered low to high.
+//! - **Presentation:** The act of proposing a candidate for insertion into the leaderboard. Presenting is
+//! `O(|number_of_voters|)`, so the presenter must be slashable and will be slashed for duplicate or invalid
+//! presentations. Presentation is only allowed during the "presentation period," after voting has closed.
+//! - **Voting bond:** Bond required to be permitted to vote. Must be held because many voting operations affect
+//! storage. The bond is held to disincentivize abuse.
+//! - **Voting:** Process of inserting approvals for oneself into storage. Can be called by anyone given they submit
+//! an appropriate list of approvals. A bond is reserved from a voter until they retract or get reported.
+//! - **Inactive voter**: A voter whose approvals are now invalid
+//!   (e.g. pointing to candidates who no longer exist).
+//!   Such voters can be _reaped_ by other voters after an `inactivity_grace_period` has passed from their last known
+//! activity.
+//! - **Reaping process:** Voters may propose the removal of inactive voters, as explained above. If the claim is not
+//! valid, the _reporter_ will be slashed and removed as a voter. Otherwise, the _reported_ voter is removed. A voter
+//! always gets his or her bond back upon being removed (either through _reaping_ or _retracting_).
+//!
+//! ### Goals
+//!
+//! The Council module in Substrate is designed to make the following possible:
+//!
+//! - Create council proposals by councillors using the council motion mechanism.
+//! - Validate council proposals.
+//! - Tally votes of council proposals by councillors during the proposal's voting period.
+//! - Veto (postpone) council proposals for a cooloff period through abstention by councillors.
+//! - Elevate council proposals to start a public referendum.
+//! - Execute referenda once their vote tally reaches the vote threshold level of approval.
+//! - Manage candidacy, including voting, term expiration, and punishment.
+//!
+//! ## Interface
+//!
+//! ### Dispatchable Functions
+//!
+//! The dispatchable functions in the Council module provide the functionality that councillors need.
```

Comment:
```suggestion
//! The dispatchable functions in the Council Module provide the functionality that councillors need.
```

Reply:
We have not been doing this. Council module.

Reply:
Ok, we'll close this then